nodepool/files/nodepool.yml

```diff
@@ -174,9 +185,20 @@ providers:
   - name: pubcloud-dfw
     region-name: 'DFW'
     cloud: public_cloud
-    boot-timeout: 600
+    # Higher boot-timeout value that for other providers to support snapshots
+    boot-timeout: 1800
     hostname-format: nodepool-{label.name}-{provider.name}-{node.id}
-    cloud-images: *provider_cloudimage_block
+    cloud-images:
+      - name: ubuntu-bionic-onmetal
+        image-name: "OnMetal - Ubuntu 18.04 LTS (Bionic Beaver)"
+      - name: ubuntu-xenial-onmetal
+        image-name: "OnMetal - Ubuntu 16.04 LTS (Xenial Xerus)"
+      - name: rpc-r14-xenial_loose_artifacts-swift-image
+        image-name: rpc-r14.17.0-xenial_loose_artifacts-swift
```

Comment:
I have no idea whether ``config-drive: true`` needs to be set for each of these new images. We do currently make use of config drive when creating them for jenkins. Perhaps it's safer to set it for now?